api/db-cdr/generate-cdr/make-bq-criteria-tables.sh

```diff
@@ -1412,7 +1412,7 @@ bq --quiet --project_id=$BQ_PROJECT query --nouse_legacy_sql \
     )
 SELECT
     (SELECT MAX(id) FROM \`$BQ_PROJECT.$BQ_DATASET.cb_criteria\`)+1 AS id
-    , 0
+    , -1
```

Comment:
So changing these parentIds from 0 to -1 will break API calls that expect the parent id to be 0. I attached a screenshot of an example UI call that is using parentId

![Screen Shot 2021-04-22 at 1 24 01 PM](https://user-images.githubusercontent.com/3904919/115766814-032ddf00-a36e-11eb-9954-ff4f92716d5a.png)

Reply:
That is correct, should be okay once this PR goes through: https://github.com/all-of-us/workbench/pull/4856